c/uca/sifter/unisifeggs.cpp

```diff
@@ -0,0 +1,440 @@
+                            unicode_1_name,
+                            iso_comment,
+                            suc,
+                            slc,
+                            stc] = fields<15>(line);
+                char32_t const code_point = parse_hex_codepoint(cp);
+                // TODO(egg): Handle ranges.
+                auto const range =
+                    CodePointRange::Inclusive(code_point, code_point);
+                coarse_general_category_[gc.front()].Include(range);
+                general_category_[std::string(gc)].Include(range);
+                canonical_combining_class_.emplace(code_point,
+                                                   std::stoi(std::string(ccc)));
+                if (!suc.empty()) {
+                    simple_uppercase_mapping_.emplace(code_point,
+                                                      parse_hex_codepoint(suc));
+                }
+                if (!slc.empty()) {
+                    simple_lowercase_mapping_.emplace(code_point,
+                                                      parse_hex_codepoint(slc));
+                }
+            }
+        }
+        ReadMultiPropertyFile(ucd_directory / "PropList.txt");
+        ReadMultiPropertyFile(ucd_directory / "DerivedCoreProperties.txt");
+        {
+            std::ifstream derived_numeric_value(ucd_directory / "extracted" /
+                                                "DerivedNumericValues.txt");
+            CHECK(derived_numeric_value.good());
+            std::cout << "Reading DerivedNumericValues.txt...\n";
+            for (std::string line; std::getline(derived_numeric_value, line);) {
+                // See https://www.unicode.org/reports/tr44/#UnicodeData.txt.
+                auto const record = strip_comment(line);
+                if (record.empty()) {
+                    continue;
+                }
+                auto const [cp, decimal_nv, _, rational_nv] = fields<4>(record);
+                auto const range = parse_hex_codepoint_range(cp);
+                code_points_with_numeric_value_.Include(range);
+                if (rational_nv.find_first_not_of("0123456789") ==
+                    std::string_view::npos) {
+                    for (char32_t const code_point : range) {
+                        natural_numeric_value_.emplace(
+                            code_point, std::stoll(std::string(rational_nv)));
+                    }
+                }
+            }
+        }
+    }
+
+    CodePointSet const& GeneralCategorySet(std::string_view const gc) {
+        if (gc.size() == 1) {
+            return coarse_general_category_.at(gc.front());
+        }
+        auto it = general_category_.find(gc);
+        CHECK(it != general_category_.end()) << gc;
+        return it->second;
+    }
+
+    CodePointSet const& BinaryPropertySet(
+        std::string_view const binary_property_name) {
+        auto it = binary_properties_.find(binary_property_name);
```

Comment:
```suggestion
    CodePointSet const& BinaryPropertySet(
            std::string_view const binary_property_name) {
        auto it = binary_properties_.find(binary_property_name);
```
... or put the whole function header on one line

Reply:
This was formatted with clang-format with Google settings; I would very much like not to fight the autoformatter.

Reply:
I guess I don't understand then. Inside Google, the formatter uses a double indent for continuing an if condition, for loop head, or function args list, so that there is a step back to the body of that statement/definition. It's hard to make out if the head continuation indent is at the same level as the body.

Reply:
Ah yes, it seems increasing the indent size to 4 spaces did not increase that to more than 4…

Reply:
I increased ContinuationIndentWidth, but that means all line continuations are very indented.